src/contracts/security/security.service.ts

```diff
@@ -109,39 +154,60 @@ export class SecurityService {
   }
 
   /**
-   * Signs a message to pause deposits with the prefix from the contract
+   * Signs a message to pause deposit contract with the prefix from the contract
    */
-  public async signPauseData(
-    blockNumber: number,
-    stakingModuleId: number,
-  ): Promise<Signature> {
+  public async signPauseDataV3(blockNumber: number): Promise<Signature> {
     const prefix = await this.repositoryService.getPauseMessagePrefix();
 
-    return await this.walletService.signPauseData({
+    return await this.walletService.signPauseDataV3({
```

Comment:
why v3?

Reply:
Just because dsmContract.VERSION contains version = 3, during Lido 2, I guess it was v2 (current version) 🤔